Assets/MinimapIcon.cs

```diff
@@ -0,0 +1,34 @@
+﻿using UnityEngine;
+
+namespace AOT.UI
```

Comment:
Namespace should match based on the location of the class. Additionally, don't place them inside the root folder